modules/manually-configure-iam-nutanix.adoc

```diff
@@ -37,16 +37,27 @@ credentials:
 <2> Specify the Prism Central credentials.
 <3> Optional: Specify the Prism Element credentials.
 
+. Set a `$RELEASE_IMAGE` variable with the release image from your installation file by running the following command:
++
+[source,terminal]
+----
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
+----
+
 . Extract the list of `CredentialsRequest` custom resources (CRs) from the {product-title} release image by running the following command:
 +
 [source,terminal]
 ----
-$ oc adm release extract --credentials-requests --cloud=nutanix \//
---to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
-quay.io/<path_to>/ocp-release:<version>
+$ oc adm release extract \
+  --from=$RELEASE_IMAGE \
+  --credentials-requests \
+  --cloud=nutanix \
+  --install-config=<path_to_directory_with_installation_configuration>/install-config.yaml \// <1>
```

Comment:
I believe `--install-config` is only needed/used in conjunction with `--included`.

Reply:
yes good catch!

Reply:
And, presumably not available in 4.13?